server/dev/server.go

```diff
@@ -69,10 +70,13 @@ func main() {
 	if err != nil {
 		server.BootLogFatalf("reading config: %v", err)
 	}
-	logger := logger.NewSimpleLogger(os.Stderr, "debug")
+	logger := logger.NewSimpleLogger(os.Stderr, "info")
+	// Setup statistics
+	currentStats := statistics.NewStatistics(logger)
+	go currentStats.PrintStats()
```

Comment:
It would be cleaner to move this goroutine inside the NewStatistics constructor. It's more of an implementation detail that is better hidden inside the statistics package.

Reply:
Ok yes agreed.